packages/dropdowns.next/src/views/combobox/StyledHint.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+import { Hint } from '@zendeskgarden/react-forms';
+
+const COMPONENT_ID = 'dropdowns.hint';
```

Comment:
#thought how do we feel about naming notation and directory structure? Should elements namespaced with `*.combobox.*` (eg `dropdowns.combobox.*`) live exclusively under the directory `src/views/combobox`?

Reply:
Agreed. We could do with a more formalized/documented taxonomy throughout. But I'll update here.